models/model_upload/test-upload/pythonic_string_cat/1/model.py

```diff
@@ -0,0 +1,44 @@
+import time
```

Comment:
since there is only one example here, do all the other examples still work with the new interface? Do we have client side examples provided for each example so that we can see how to call them with the old interface and the new interface examples with the new interface?